src/modularity.jl

```diff
@@ -1,73 +1,128 @@
 """
-    randompartition(hg::Hypergraph, n::Int)::Vector{Vector{Int}}
 
-Generates a random partition for vertices of a hypergraph `hg` into `n` subsets.
+    randompartition(h::Hypergraph, n::Int)::Vector{Set{Int}}
```

Comment:
something is mixed up here

Reply:
looks OK in the source code.

Reply:
Sorry - I meant the one below. The docstring function signature does not math the signature in code.